core/src/components/cat-datepicker/cat-datepicker.tsx

```diff
@@ -249,9 +264,8 @@ export class CatDatepickerFlat {
         onCatChange={e => {
           e.stopPropagation();
           this.pickr?.setDate(e.detail);
```

Comment:
if we are setting the value below and the watch kicks in, shouldnt this line be redundant and be deleted?